main.py

```diff
@@ -411,12 +418,92 @@ def perplexity_eval(model, testenc, args, dev):
 
     get_model_head(model).to(torch.device("cpu"))
 
-    if args.wandb:
-        wandb.log({args.dataset_name: ppl.item()})
-
     model.config.use_cache = use_cache
 
 
+def generate(model, dev, model_path):
```

Comment:
what this function for and why is it in main.py?

Reply:
There was no need for me to commit any changes to main.py.

Resolved by reverting everything back.